src/backends/pgfplotsx.jl

```diff
@@ -325,41 +305,34 @@ function (pgfx_plot::PGFPlotsXPlot)(plt::Plot{PGFPlotsXBackend})
             else
                 push!(plt.o, the_plot)
             end
-        end # for subplots
+        end  # for subplots
         pgfx_plot.is_created = true
         pgfx_plot.was_shown = false
-    end # if
+    end
     return pgfx_plot
 end
 ## seriestype specifics
-function pgfx_add_series!(axis, series_opt, series, series_func, opt)
-    args = pgfx_series_arguments(series, opt)
-    series_plot = series_func(series_opt, PGFPlotsX.Table(args...))
-    push!(axis, series_plot)
-    pgfx_add_legend!(axis, series, opt)
-end
+pgfx_add_series!(axis, series_opt, series, series_func, opt) = pgfx_add_legend!(
+    push!(axis, series_func(series_opt, Table(pgfx_series_arguments(series, opt)...))),
+    series,
+    opt,
+)
 
 function pgfx_add_series!(::Val{:path}, axis, series_opt, series, series_func, opt)
     # treat segments
     segments = collect(series_segments(series, series[:seriestype]; check = true))
     sf = opt[:fillrange]
     for (k, segment) in enumerate(segments)
         i, rng = segment.attr_index, segment.range
-        segment_opt = PGFPlotsX.Options()
-        segment_opt = merge(segment_opt, pgfx_linestyle(opt, i))
+        segment_opt = merge(Options(), pgfx_linestyle(opt, i))
         if opt[:markershape] !== :none
-            marker = _cycle(opt[:markershape], i)
```

Comment:
> I also believe that the if (val = something) == whatever construct, akin to Python 's walrus operator := should be preferred

We can agree to disagree here, I think needing to scan arbitrary expressions for variable definitions instead of just line beginnings, when looking for where something comes from is a real pain.